src/Tasks/Microsoft.NET.props

```
@@ -7,13 +7,22 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
           impossible to load or build your projects from the command-line or the IDE.
 
 This file contains .NET-specific properties, and items. This file is imported for .NET Core, .NET Standard, and .NET Framework projects.
-these two files are used to encapsulate the multi-targeting and framework specific build process.
 
 Copyright (C) Microsoft Corporation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project>
 
+  <!--
+    ============================================================
+                                        Reference Assemblies
+    Enable the production of a reference assembly by all .NET projects, by default.
+    ============================================================
+  -->
+  <PropertyGroup>
+    <ProduceReferenceAssembly Condition=" '$(ProduceReferenceAssembly)' == '' ">true</ProduceReferenceAssembly>
```

`ProduceReferenceAssembly` and `ProduceOnlyReferenceAssembly` are mutually exclusive, so I think we need to tweak this a bit:

```suggestion
    <ProduceReferenceAssembly Condition=" '$(ProduceReferenceAssembly)' == '' and '$(ProduceOnlyReferenceAssembly)' != 'true' ">true</ProduceReferenceAssembly>
```